jni/Android.mk

```diff
@@ -218,4 +218,20 @@ LOCAL_C_INCLUDES += $(GST_HEADERS_COMMON)
 
 include $(BUILD_SHARED_LIBRARY)
 
+include $(CLEAR_VARS)
+LOCAL_MODULE := nnstreamer_decoder_octet_stream
+LOCAL_SRC_FILES := $(NNSTREAMER_DECODER_PE_SRCS)
```

Comment:
update nnstreamer.mk to define NNSTREAMER_DECODER_OS_SRCS

Reply:
Thanks. Updated.